EFTools.msbuild

```diff
@@ -90,20 +93,35 @@
     </Target>
 
     <Target Name="BuildTestHost">
-        <MSBuild Targets="Build"
+        <Message Condition="'$(IsDev14OrLower)' != 'True'"
```

Comment:
Actually the messages are slightly different. Also it draws attention to the Condition in the next line. So don't think this is worth it.